util/jsonresponse.py

```diff
@@ -117,12 +122,18 @@ def _particles(self, stream_data, stream_key, parameters, external_includes):
 
             for index in xrange(len(ds.time)):
                 # Create our particle from the list of parameters
-                particle = {p: data[p][index] for p in params}
+                particle = {}
+                for p in params:
+                    if p in ds and 'obs' not in ds[p].dims:
+                        # data has no obs dimension (13025 AC2)
+                        particle[p] = data[p]
+		    else:
+                        # data is bound by obs dimensino
```

Comment:
`dimension`

Reply:
hah, I kinda like how this mispelling sounds though.  i'll get it fixed.